specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -2462,14 +2512,22 @@
           "type": "string",
           "description": "The identity type.",
           "enum": [
-            "SystemAssigned"
+            "SystemAssigned",
+			      "UserAssigned"
```

Comment:
Currently, we can only input one of SystemAssigned, UserAssigned
But per the [rest Spec](https://microsoft.sharepoint.com/:w:/t/AzureStorage/EZDFILSkeoVLgHFyCBnIALsBPV6sgn72-zD5xIxULpn5bQ?e=kek5iP&CID=85117BC7-29BC-49AB-B954-08438E6D7711&wdLOR=c3A327EC9-6BD5-41F0-BBAE-AF85C2650DF5)， the valid value is: `One of [‘SystemAssigned’, ‘UserAssigned’, ‘SystemAssigned,UserAssigned’, 
‘None’]. ` Please update the swagger to be aligned.

Reply:
This needs to be updated to [‘SystemAssigned’, ‘UserAssigned’, ‘SystemAssigned,UserAssigned’, ‘None’]
as Wei Wei has suggested.

Reply:
Ack

Reply:
Fixed it